src/audio/mixer.c

```diff
@@ -464,22 +456,50 @@ static int mixer_prepare(struct comp_dev *dev)
 			return PPL_STATUS_PATH_STOP;
 	}
 
+	return 0;
+}
+
+#if CONFIG_IPC_MAJOR_3
+static int mixer_trigger(struct comp_dev *dev, int cmd)
```

Comment:
Can we state the differences as a comment between IPC3/4 so that every can follow why we do things differently.